strawberry/codegen/types.py

```diff
@@ -32,6 +34,7 @@ class GraphQLField:
     alias: Optional[str]
     type: GraphQLType
     default_value: Optional[GraphQLArgumentValue] = None
+    definition_node: Optional[Node] = None
```

Comment:
do you think it would be possible to use the strawberry type here instead?

Reply:
We could do both...

The query is the part that is controlled by the client (they may not control the upstream strawberry type).  I actually have a branch that I haven't quite gotten working that tries to build a client from *only* a graphql schema (then the Python client could be generated against things that don't even use strawberry as the graphql provider).

But... It would be nice to have access to the strawberry type where available.  That's probably easier to work with than digging through the graphql AST...